plugins/tiledb/io/TileDBReader.cpp

```diff
@@ -295,7 +295,7 @@ void TileDBReader::localReady()
     // read spatial reference
     NL::json meta = nullptr;
 
-#if TILEDB_VERSION_MAJOR >= 1 && TILEDB_VERSION_MINOR >= 7
+#if TILEDB_VERSION_MAJOR > 1 || TILEDB_VERSION_MINOR >= 7
```

Comment:
This doesn't seem robust going forward.

Reply:
In what way? If the version of TileDB is greater than 1.7 then use internal TIleDB metadata.

Reply:
It's fine.  Sorry.